src/Enzo/tests/EnzoInitialFeedbackTest.cpp

```diff
@@ -143,32 +164,40 @@ void EnzoInitialFeedbackTest::enforce_block
   int ngz = nz + 2*gz;
 
   const enzo_float gamma = enzo::fluid_props()->gamma();
+  // shouldn't we be using a self-consistent mmw?
   const enzo_float mol_weight = enzo::fluid_props()->mol_weight();
```

Comment:
I assume this is reading the molecular weight from somewhere else, like another parameter? Maybe make this an issue so we don't loose track of it?

Reply:
Yeah, users have the option to specify a single,fixed mmw weight by using the parameter `Physics:fluid_props:mol_weight`. I added the comment because we go on and subsequently use various species fields. 

It doesn't appear that the mmw computed from the species fields will be consistent with the mmw specified in the parameters.

(I'll add this note to the species issue)